front/src/modules/companies/components/AddPersonToCompany.tsx

```diff
@@ -0,0 +1,109 @@
+import { useState } from 'react';
+import { getOperationName } from '@apollo/client/utilities';
+import styled from '@emotion/styled';
+import { autoUpdate, flip, offset, useFloating } from '@floating-ui/react';
+import { IconPlus } from '@tabler/icons-react';
+
+import {
+  PeoplePicker,
+  PersonForSelect,
+} from '@/people/components/PeoplePicker';
+import { GET_PEOPLE } from '@/people/graphql/queries/getPeople';
+import {
+  Button,
+  ButtonSize,
+  ButtonVariant,
+} from '@/ui/button/components/Button';
+import { RecoilScope } from '@/ui/utilities/recoil-scope/components/RecoilScope';
+import { useIsMobile } from '@/ui/utilities/responsive/hooks/useIsMobile';
+import {
+  useGetPeopleQuery,
+  useUpdateOnePersonMutation,
+} from '~/generated/graphql';
+
+const StyledButton = styled(Button)`
+  &:focus {
+    color: ${({ theme }) => theme.grayScale.gray40};
+  }
+`;
+
+const StyledDropdownContainer = styled.div<{ isMobile: boolean }>`
+  left: unset !important;
+  right: ${({ isMobile }) => (isMobile ? '0' : 'unset')};
+  top: 35px !important;
+`;
+
+const StyledContainer = styled.div`
+  position: relative;
+`;
+
+export function AddPersonToCompany({ companyId }: { companyId: string }) {
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+  const [updatePerson] = useUpdateOnePersonMutation();
+  const isMobile = useIsMobile();
+  const { refs, floatingStyles } = useFloating({
+    strategy: 'absolute',
+    middleware: [offset(10), flip()],
+    whileElementsMounted: autoUpdate,
+    placement: 'right-start',
+  });
+
+  const { data: { people: peopleWithoutCompany } = { people: [] } } =
+    useGetPeopleQuery({
+      variables: {
+        orderBy: [],
+        where: {
+          company: null,
+        },
+      },
+    });
+
+  function handlePersonSelected(companyId: string) {
+    return async (newPerson: PersonForSelect | null) => {
+      if (newPerson) {
+        await updatePerson({
+          variables: {
+            where: {
+              id: newPerson.id,
+            },
+            data: {
+              company: { connect: { id: companyId } },
+            },
+          },
+          refetchQueries: [getOperationName(GET_PEOPLE) ?? ''],
+        });
+      }
+    };
+  }
+
+  function handleCancel() {
+    if (isDropdownOpen) setIsDropdownOpen(false);
+  }
+
+  return (
+    <RecoilScope>
+      <StyledContainer>
+        <StyledButton
+          icon={<IconPlus size={14} />}
+          onClick={() => setIsDropdownOpen(!isDropdownOpen)}
+          size={ButtonSize.Small}
+          variant={ButtonVariant.Tertiary}
+        />
+
+        {isDropdownOpen && peopleWithoutCompany.length > 0 && (
```

Comment:
we should not have peopleWithoutCompany.length > 0 condition as:
- we allow to select person from other companies
- if there is nobody that is not in the team, we would just display an empty dropdown (which should almost never happen because nobody will only have one company in their CRM)

We need to exclude people that are already attached to the company from the dropdown though. I think we need to pass an additional excludePersonIds to PeoplePicker

Also, I've tested it and clicking on a person should add it to the team AND close the dropdown